### PR TITLE
Fix install and script bugs

### PR DESCRIPTION
**Pull request by NeoZhangJianyu**
Fix install and script bugs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,20 +22,20 @@ Step-by-step Deep Learning Tutorials on Apache Spark using [BigDL](https://githu
...
++ Apache Spark >= 2.2.1
 + Jupyter Notebook 4.1
-+ BigDL 0.5.0
++ BigDL 0.7.0
 + [Setup env on Mac OS](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupMac.md) / [Setup env on Linux](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupLinux.md)
 
 ### Start Jupyter Server
-* Run ```pip install BigDL==0.5.0```
-* Run ```jupyter notebook --notebook-dir=./ --ip=* --no-browser```
+* Run ```sudo pip install BigDL==0.7.0```
```
Comment (i8run) [inline README.md]: I think we should not use sudo to install the packages under `/usr/local/`. Could we use the `PYTHONPATH` to handle privileges ?

File: README.md [hunk 2]
```diff
@@ -22,20 +22,20 @@ Step-by-step Deep Learning Tutorials on Apache Spark using [BigDL](https://githu
...
+* Run ```jupyter notebook --notebook-dir=./ --ip=0.0.0.0 --no-browser```
 
 ### Start Toree Kernel to Run Scala Notebooks
-* Run ```pip install BigDL==0.5.0```
-* Run ```pip install https://dist.apache.org/repos/dist/dev/incubator/toree/0.2.0/snapshots/dev1/toree-pip/toree-0.2.0.dev1.tar.gz```
-* Run ```./toree_install.sh```
-* Run ```jupyter notebook --notebook-dir=./ --ip=* --no-browser```
+* Run ```sudo pip install BigDL==0.7.0```
+* Run ```sudo pip install https://dist.apache.org/repos/dist/release/incubator/toree/0.2.0-incubating/toree-pip/toree-0.2.0.tar.gz```
+* Run ```sudo ./toree_install.sh```
```
Comment (i8run) [inline README.md]: The same as above.

File: README.md (hunk 1 above)
Comment (NeoZhangJianyu) [reply README.md]: Yes. Accept it.
I will remove 'sudo'

File: README.md (hunk 2 above)
Comment (NeoZhangJianyu) [reply README.md]: ./toree_install.sh meet error, which could be fixed by add 'sudo'.
Do you know if there any other methods to fix it?

./toree_install.sh 
Install Toree
[ToreeInstall] Installing Apache Toree version 0.2.0
[ToreeInstall] 
Apache Toree is an effort undergoing incubation at the Apache Software
Foundation (ASF), sponsored by the Apache Incubator PMC.

Incubation is required of all newly accepted projects until a further review
indicates that the infrastructure, communications, and decision making process
have stabilized in a manner consistent with other successful ASF projects.

While incubation status is not necessarily a reflection of the completeness
or stability of the code, it does indicate that the project has yet to be
fully endorsed by the ASF.
[ToreeInstall] Creating kernel Scala
[ToreeInstall] Removing existing kernelspec in /usr/local/share/jupyter/kernels/apache_toree_scala
Traceback (most recent call last):
  File "/usr/local/bin/jupyter-toree", line 11, in <module>
    load_entry_point('toree==0.2.0', 'console_scripts', 'jupyter-toree')()
  File "/usr/local/lib/python3.6/dist-packages/toree/toreeapp.py", line 167, in main
    ToreeApp.launch_instance()
  File "/usr/lib/python3/dist-packages/traitlets/config/application.py", line 658, in launch_instance
    app.start()
  File "/usr/local/lib/python3.6/dist-packages/toree/toreeapp.py", line 164, in start
    return self.subapp.start()
  File "/usr/local/lib/python3.6/dist-packages/toree/toreeapp.py", line 131, in start
    replace=self.replace
  File "/usr/lib/python3/dist-packages/jupyter_client/kernelspec.py", line 344, in install_kernel_spec
    shutil.rmtree(destination)
  File "/usr/lib/python3.6/shutil.py", line 480, in rmtree
    _rmtree_safe_fd(fd, path, onerror)
  File "/usr/lib/python3.6/shutil.py", line 438, in _rmtree_safe_fd
    onerror(os.unlink, fullname, sys.exc_info())
  File "/usr/lib/python3.6/shutil.py", line 436, in _rmtree_safe_fd
    os.unlink(name, dir_fd=topfd)
PermissionError: [Errno 13] Permission denied: 'kernel.json'

**Review by i8run — APPROVED** [review]

**Pull request merged**